Training_25/Program.cs

```diff
@@ -3,11 +3,49 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T01 branch - Assignments Q1: NUMBER CONVERSION GAME
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
+using static System.Math;
 namespace Training_25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      WriteLine ("Enter a number");
+      string? input = ReadLine ();
+      int num;
+      while (string.IsNullOrWhiteSpace (input) || !int.TryParse (input, out num)) {
+         WriteLine ("Invalid input. Please enter a  number:");

```

Comment:
```suggestion
         WriteLine ("Invalid input. Please enter a number:");
```
Remove the additional space before `number`.